src/metabase/server/middleware/session.clj

```diff
@@ -65,29 +74,28 @@
   [request response {session-uuid :id} :- {:id (s/cond-pre UUID u/uuid-regex), s/Keyword s/Any}]
   (let [response       (wrap-body-if-needed response)
         cookie-options (merge
-                        {:same-site config/mb-session-cookie-samesite
-                         :http-only true
-                         ;; TODO - we should set `site-path` as well. Don't want to enable this yet so we don't end
-                         ;; up breaking things
-                         :path      "/" #_ (site-path)}
-                        ;; If the env var `MB_SESSION_COOKIES=true`, do not set the `Max-Age` directive; cookies
-                        ;; with no `Max-Age` and no `Expires` directives are session cookies, and are deleted when
-                        ;; the browser is closed
-                        ;;
-                        ;; See https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Session_cookies
-                        (when-not (config/config-bool :mb-session-cookies)
-                          ;; max-session age-is in minutes; Max-Age= directive should be in seconds
-                          {:max-age (* 60 (config/config-int :max-session-age))})
-                        ;; If the authentication request request was made over HTTPS (hopefully always except for
-                        ;; local dev instances) add `Secure` attribute so the cookie is only sent over HTTPS.
-                        (when (request.u/https? request)
-                          {:secure true})
-                        (when (= config/mb-session-cookie-samesite :none)
-                          (log/warn
-                           (str (deferred-trs "Session cookie's SameSite is configured to \"None\", but site is")
-                                (deferred-trs "served over an insecure connection. Some browsers will reject ")
-                                (deferred-trs "cookies under these conditions. ")
-                                (deferred-trs "https://www.chromestatus.com/feature/5633521622188032")))))]
+                         {:same-site config/mb-session-cookie-samesite
```

Comment:
The old indentation for `merge` was actually correct. `merge` is a function call, so if its arguments are all on new lines, they should be indented 1 space. 

2 spaces is for macros or special forms with a varargs `& body`, which includes things like `let` or `do` or `when` or `def`.

Your editor should do this for you automatically tho. Since `merge` is a core function it shouldn't even need to have an active nREPL connection to know the correct indentation. I'm not sure how to configure VIM do this correctly, but maybe @walterl can point you in the right direction

Reply:
This is actually one of the things that I haven't managed to figure out yet. Please let me know if you find a solution.

Reply:
@walterl I think I just got this behavior working by installing this plugin: https://github.com/guns/vim-clojure-static and putting `let g:clojure_align_subforms = 1` in my vimrc. Not sure if there are any unintended side effects of this plugin yet...

Reply:
Ah yes. I had that in my config for a while too, but it only fixes half the use cases: it either formats _all_ forms with one space (enabled), or _all_ forms with 2 spaces (disabled). The [single space indent guildeline](https://guide.clojure.style/#one-space-indent) is for _function calls_ with no args on the same line as the function. Macros taking a `& body` (e.g. `do`) should still be indented 2 spaces.

Reply:
Ah I see. Yeah, seems like `do` doesn't indent properly now.

Reply:
Here's a workaround to make specific words indent by two space: https://github.com/guns/vim-clojure-static#gclojure_fuzzy_indent-gclojure_fuzzy_indent_patterns-gclojure_fuzzy_indent_blacklist

`autocmd FileType clojure set lispwords+=do` makes `do` work properly for me.

Reply:
Great! I don't think I've tried that one before. :raised_hands: